mk/i386-mingw32-gcc.mk

```diff
@@ -1,13 +1,14 @@
 ifeq (${_INCLUDE_MK_GCC_},)
 _INCLUDE_MK_GCC_=1
-# osx port install mingw32-gcc
-CC=i386-mingw32-gcc
-RANLIB=i386-mingw32-ranlib
+CROSS=i386-mingw32-
+CC=${CROSS}gcc
+RANLIB=${CROSS}ranlib
 OSTYPE=windows
 ONELIB=0
 LINK=-shared
-AR=i386-mingw32-ar
-CC_AR=$(AR) -r ${LIBAR}
+AR=${CROSS}ar
+CC_AR=${AR} -r ${LIBAR}
+PARTIALLD=${CROSS}ld -r --whole-archive
```

Comment:
why not name it CC_LD or just LD?

Reply:
Because it's a partial linker invocation specifically (i.e with `-r` flag and sometimes `--whole-archive`), not a general one, you cannot link an executable or a library with it.